br/pkg/lightning/config/config.go

```diff
@@ -853,13 +855,15 @@ func NewConfig() *Config {
 			DataInvalidCharReplace: string(defaultCSVDataInvalidCharReplace),
 		},
 		TikvImporter: TikvImporter{
-			Backend:             "",
-			OnDuplicate:         ReplaceOnDup,
-			MaxKVPairs:          4096,
-			SendKVPairs:         32768,
-			RegionSplitSize:     0,
-			DiskQuota:           ByteSize(math.MaxInt64),
-			DuplicateResolution: DupeResAlgNone,
+			Backend:                "",
+			OnDuplicate:            ReplaceOnDup,
+			MaxKVPairs:             4096,
+			SendKVPairs:            32768,
+			RegionSplitSize:        0,
+			RegionSplitBatchSize:   4096,
```

Comment:
The default value 4096 is much too big, how about a little lower value such as 128 ?